autox/k8sclient.go

```diff
@@ -7,16 +7,10 @@ import (
 
 	"helm.sh/helm/v3/pkg/cli"
 
-	// Import to initialize client auth plugins.
-	_ "k8s.io/client-go/plugin/pkg/client/auth"
-
```

Comment:
Why was this import removed? while it isn't needed more than once, it is essential when using any cloud with authentication

Reply:
I am not sure why this was deleted. Maybe it was an unintended mistake. I will restore it.